core/controllers/feedback.py

```diff
@@ -64,6 +64,20 @@ def post(self, exploration_id):
         self.render_json(self.values)
 
 
+class ThreadListHandlerForTopics(base.BaseHandler):
+    """Handles listing of feedback threads linked to topics."""
+    @acl_decorators.can_edit_topic
+    def get(self, topic_id):
+        # TODO (nithesh): Add non-suggestion feedback threads when required in
```

Comment:
I think you can drop this TODO. If it's required we'll naturally add it.

Reply:
Done